### PR TITLE
Wizards: Update notice-like elements with the Notice component

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

PR removes the custom CSS/SVG and use the Notice component instead.

Wizards:
* Health Check
* Performance
* Reader Revenue
* Configure Plugins
* Error Modal (via With Wizards)

### How to test the changes in this Pull Request:

1. Switch to this branch.
2. Check the different pages.
3. Text + icon should be using the `Notice` component

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: assets/wizards/readerRevenue/views/configure-landing-page/index.js [hunk 1]
```diff
@@ -25,33 +23,15 @@ class ConfigureLandingPage extends Component {
...
 				{ donationPage && (
 					<Fragment>
 						{ 'publish' !== donationPage.status && (
-							<div className="newspack-memberships-page-wizard-wizard__notice">
-								{ notPublishedIcon }
-								{ __(
-									'Your donations landing page has been created, but is not yet published. You can now edit it and publish when you\'re ready.'
-								) }
-							</div>
+							<Notice isError noticeText={ __( 'Your donations landing page has been created, but is not yet published. You can now edit it and publish when you\'re ready.' ) } />
```
Comment (jeffersonrabb) [inline assets/wizards/readerRevenue/views/configure-landing-page/index.js]: There's inconsistency between Membership and Donation in these notices and buttons. It's unrelated to this PR, so probably should be resolved in a separate oone.

File: assets/wizards/performance/views/intro/index.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ class Intro extends Component {
 					image={ settings.site_icon }
 					onChange={ image => updateSiteIcon( image ) }
 				/>
-				<p><em>{ __( 'Site icons should be square and at least 144 × 144 pixels.' ) }</em></p>
+				<Notice noticeText={ __( 'Site icons should be square and at least 144 × 144 pixels.' ) } isPrimary />
```
Comment (jeffersonrabb) [inline assets/wizards/performance/views/intro/index.js]: Just confirming that `isPrimary` is meant to render the icon in blue as opposed to red?

Comment (thomasguillot) [reply assets/wizards/performance/views/intro/index.js]: Yep, blue.

File: assets/wizards/readerRevenue/views/configure-landing-page/index.js (hunk 1 above)
Comment (thomasguillot) [reply assets/wizards/readerRevenue/views/configure-landing-page/index.js]: I agree, it should be resolved in a separate PR.

**Pull request merged**